include/asterisk/channel.h

```diff
@@ -4368,6 +4368,24 @@ struct ast_str *ast_channel_dialed_causes_channels(const struct ast_channel *cha
  */
 struct ast_control_pvt_cause_code *ast_channel_dialed_causes_find(const struct ast_channel *chan, const char *chan_name);
 
+/*!
+ * \since 22
```

Comment:
I don't think \since 22 is correct here, someone else may have further comments on this